### PR TITLE
Clarify note on creating named pipe

### PR DESCRIPTION
**Pull request by zanedp**
As it was, it has misled many engineers down an unnecessary and confusing path.

### CODE REVIEW

File: docset/winserver2022-ps/hyper-v/Set-VMComPort.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ PS C:\> Set-VMComPort TestVM 2 \\.\pipe\TestPipe
 Configure the second COM port on virtual machine TestVM to connect to named pipe TestPipe on the local computer.
 
 > [!NOTE]
-> This does not create the named pipe on the local computer. You will have to write your own application to create a named pipe.
+> You can write write your own application to access the named pipe, or use an application like PuTTY.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/hyper-v/Set-VMComPort.md:50]:
```suggestion
> You can write your own application to access the named pipe, or use an application like PuTTY.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @zanedp, kindly commit the suggested change to correct the typo. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/hyper-v/Set-VMComPort.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ PS C:\> Set-VMComPort TestVM 2 \\.\pipe\TestPipe
 Configure the second COM port on virtual machine TestVM to connect to named pipe TestPipe on the local computer.
 
 > [!NOTE]
-> This does not create the named pipe on the local computer. You will have to write your own application to create a named pipe.
+> You can write your own application to access the named pipe, or use an application like PuTTY.
```
Comment (dariomws) [inline docset/winserver2022-ps/hyper-v/Set-VMComPort.md:50]:
```suggestion
> This does not create the named pipe on the local computer. You can write your own application to access the named pipe, or use a third-party application.
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by get-itips — CHANGES_REQUESTED** [review]

**Comment by get-itips** [comment]
@zanedp please choose one of the two suggestions to replace actual application names, thank you!

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**